config/god/hets_workers.rb

```diff
@@ -11,8 +11,13 @@ def start_cmd
   end
 
   def hets_server_options
-    yml_file = File.join(RAILS_ROOT, 'config', 'hets.yml')
-    YAML.load_file(yml_file)['server_options'] || []
+    if ! defined?(AppConfig)
```

Comment:
Style/NegatedIf: Favor `unless` over `if` for negative conditions.<br>Style/SpaceAfterNot: Do not leave space between `!` and its argument.